pyaedt/modules/Material.py

```diff
@@ -675,6 +685,38 @@ def __init__(self, parent, name, props=None):
                                                             MatProperties.get_defaultvalue(aedtname=property), None)
         pass
 
+    @property
+    def material_appearance(self):
+        """Material Appearance specified as an RGB list.
+
+        Returns
+        -------
+        type
+            Appearance of the material.
+        """
+        return self._material_appearance
+
+    @material_appearance.setter
+    def material_appearance(self, value):
+        if type(value) is list and len(value) == 3:
```

Comment:
Raising a `ValueError` won't be helpful here.  Let's try this

```suggestion
        if not isinstance(value, (list, tuple)):
            raise TypeError('`material_apperance` must be a list or tuple)
        if len(value) != 3:
            raise ValueError('`material_appearance` must be three items (RGB)')
```

Reply:
You are fully right. Much better.